qa/rpc-tests/mempool_packages.py

```diff
@@ -110,7 +114,74 @@ def run_test(self):
                 assert_equal(i, 999)
                 print("tx that would create too large descendant package successfully rejected")
 
+        # TODO: check that node1's mempool is as expected
```

Comment:
Were these TODOs ever done?

Reply:
Yes, much later on in https://github.com/bitcoin/bitcoin/pull/17435. It looks like a relatively clean backport that I can try after this chain of PRs merges.

Reply:
(So put another way, no this TODO was not ever done, because that PR did not implement all of the checks that this TODO implied, and it left its own TODO.)